service/build.gradle

```diff
@@ -191,6 +191,11 @@ dependencies {
   compile group: 'org.springframework.security', name: 'spring-security-oauth2-resource-server'
   compile group: 'org.springframework.security', name: 'spring-security-oauth2-jose'
   compile group: 'com.nimbusds', name: 'nimbus-jose-jwt', version: '8.5.1' // override to fix vulnerability issue (CVE-2019-17195)
+  compile group: 'com.squareup.okhttp3', name: 'okhttp', version: '4.3.1'
+  compile group: 'com.launchdarkly', name: 'okhttp-eventsource', version: '1.10.1'
+  compile group: 'com.launchdarkly', name: 'launchdarkly-java-server-sdk', version: '4.11.1', classifier: 'thin'
+  compile group: 'io.github.openfeign', name: 'feign-core', version: '10.4.0'
+  compile group: 'uk.gov.hmcts.reform', name: 'send-letter-client', version: '2.3.1'
```

Comment:
Thoughts of grouping this with the other reform dependencies, just a nice bit of ordering

Reply:
good point, fixed